tests/console/zypper_in.pm

```diff
@@ -56,7 +56,7 @@ sub run {
     script_run 'zypper -n in --allow-unsigned-rpm ' . data_url('zypper/pa.rpm');
     script_run 'rm -f /preinstall_fail';
 
-    assert_script_run 'zypper -n in adaptec-firmware adobe-sourcecodepro-fonts alsa-devel apache2 atmel-firmware';
+    assert_script_run 'zypper -n in adaptec-firmware geoipupdate alsa-devel apache2 atmel-firmware';
```

Comment:
https://openqa.opensuse.org/tests/2220481#step/zypper_in/26 - alsa-devel is amongst the not-always-existing packages

Reply:
or SLE test: https://openqa.suse.de/tests/8252862#step/zypper_in/26